apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -50,9 +55,37 @@ export class DotEditLayoutComponent implements OnInit {
                 this.templateContainersCacheService.set(mappedContainers);
             });
 
+        this.updateTemplate
```

Comment:
Add doc explaining why this approach and if you can the link where you found this solution.